frontend/src/App.tsx

```diff
@@ -33,6 +34,10 @@ const router = createBrowserRouter([
     path: "delete-profile",
     element: <ProfileDeletion />,
   },
+  {
+    path: "pets",
+    element: <PetList />,
```

Comment:
We should wrap these into protectedpages soon. Just FYI dont need to change now.

Reply:
I've just put it to the securityFilterChain. Below commit.

Reply:
I mean the frontend components, should be protected.

Reply:
I put it into the main.tsx to be shown if the user is authenticated. Do you mean this protection?

Reply:
Like this:
https://github.com/gfa-cc-after/Pet-Clinic-GilmoreDevs/blob/main/frontend/src/pages/utils/ProtectedPage.tsx

Reply:
Can we wrap these into protectedPages @Hsbalazs ?

Reply:
I wrapped PetList into ProtectedPages.